### PR TITLE
4826: Update endpoint for dashboard

### PR DESCRIPTION
**Pull request by kat1906**
Story: [AB#4826](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/4826)
Task: [AB#6884](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/6884)

### CODE REVIEW

File: app/pages/dashboard/ui.test.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const setCookies = ClientFunction(() => {
 
 const mocks = () => {
   nock(orderApiUrl)
-    .get('/api/v1/orders')
+    .get('/api/v1/organisation/undefined/orders')
```
Comment (alibombaye) [inline app/pages/dashboard/ui.test.js:20]: we should be able to pass in an orgId from the claim right? might need to update that fake cookie we set up to return a primaryOrganisationId

Comment (kat1906) [reply app/pages/dashboard/ui.test.js:20]: I have no idea what to do with this, tech debt?

File: testcafeRunner.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ const browserToRun = browserFromArgs.length > 0 ? browserFromArgs : 'chrome:head
 const testFromArgs = process.argv.slice(3, 4);
 const testsToRun = testFromArgs ? `**/*${testFromArgs}*/ui.test.js` : '**/*ui.test.js';
 
-let concurrency = 4;
+let concurrency = 1;
```
Comment (alibombaye) [inline testcafeRunner.js:20]: Next PR whoever gets in there first. Lets take this out. The concurrency doesn't work with the nock checks

Comment (kat1906) [reply testcafeRunner.js:20]: take out completely?

**Pull request merged**